docs/use-cases/sensor-module.md

```diff
@@ -0,0 +1,392 @@
+  "timezone": "America/Los_Angeles"
+}
+responses = openmeteo.weather_api(url, params=params)
+
+# Process first location.
+response = responses[0]
+print(f"Coordinates {response.Latitude()}°N {response.Longitude()}°E")
+print(f"Elevation {response.Elevation()} m asl")
+print(f"Timezone {response.Timezone()} {response.TimezoneAbbreviation()}")
+print(f"Timezone difference to GMT+0 {response.UtcOffsetSeconds()} s")
+
+# Current values. The order of variables needs to be the same as requested.
+current = response.Current()
+current_pm10 = current.Variables(0).Value()
+current_pm2_5 = current.Variables(1).Value()
+
+print(f"Current time {current.Time()}")
+print(f"Current pm10 {current_pm10}")
+print(f"Current pm2_5 {current_pm2_5}")
+```
+
+{{< /expand >}}
+
+{{< expand "An example of getting PM2.5 and PM10 readings over serial from sensor hardware" >}}
+This script is based on the [pms7003 module](https://app.viam.com/module/joyce/pms7003).
+Before creating the module, the following script could have been used to verify the code that gets the readings from the sensor.
+
+```python {class="line-numbers linkable-line-numbers"}
+# my-sensor-test.py
+from serial import Serial
+import time
+
+def main():
+
+    port = Serial('/dev/ttyAMA0', baudrate=9600)
+
+    def parse_data(data):
+
+        if len(data) < 2:
+            return {}
+        if data[0] == 0x42 and data[1] == 0x4d:
+            pm2_5_atm = (data[12] << 8) + data[13]
+            pm10_atm = (data[14] << 8) + data[15]
+            print(f'PM2.5 (atmospheric): {pm2_5_atm} µg/m3')
+            print(f'PM10 (atmospheric): {pm10_atm} µg/m3')
+
+            # Return a dictionary of the readings
+            return {
+                "pm2_5_atm": pm2_5_atm,
+                "pm10_atm": pm10_atm,
+            }
+
+        else:
+            print('Data does not start with the expected start bytes.')
+            return {}
+
+    while port.in_waiting == 0:
+        time.sleep(0.01)  # wait for 10 ms
+
+    data = port.read(port.in_waiting)
+    print(parse_data(data))
+
+if __name__ == "__main__":
+    main()
+```
+
+{{< /expand >}}
+
+{{< expand "Another example (this one is for a base, not a sensor)" >}}
+
+An example from the [custom robot dog base tutorial](/tutorials/custom/custom-base-dog/).
+When you run this with the robot dog hardware, you can tell it works because the robot dog takes a few steps forward.
+
+```python {class="line-numbers linkable-line-numbers"}
+# dog_test.py is for testing the connection
+import socket
+import time
+
+s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+s.connect(("PASTE DOG IP ADDRESS HERE", 5001))
+
+cmd = "CMD_MOVE_FORWARD#15"
+s.send(cmd.encode("utf-8"))
+time.sleep(7)
+cmd = "CMD_MOVE_STOP"
+s.send(cmd.encode("utf-8"))
+cmd = "CMD_RELAX"
+s.send(cmd.encode("utf-8"))
+```
+
+{{< /expand >}}
+
+Run your test script from your command line and make sure you are able to get readings from the sensor before proceeding.
```

Comment:
```suggestion
Run your test script from your command line interface and make sure you are able to get readings from the sensor before proceeding.
```

Reply:
I believe a "CLI" is something you interface with from within the terminal, such as the Viam CLI. I will change this to "terminal window" or "terminal" because that seems to be more general, whereas "command prompt" is more Windows-based.